### DISCUSSION TITLE
Using anchor links to link the clipped text to specific parts of a webpage

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#85] [Ideas]
I was experimenting by using anchor links to link the clipped text to specific parts of a webpage 

![image](https://user-images.githubusercontent.com/67494083/157113303-0b6d33db-70e7-40a4-ad03-ff8742cde49f.png)

![image](https://user-images.githubusercontent.com/67494083/157113358-d270eb76-847a-4ecf-b0f3-07516c9bcd93.png)

![image](https://user-images.githubusercontent.com/67494083/157113391-6d8dc47e-c23d-4123-bbda-ea953885417f.png)

Its working. But I have to manually find the `id `of the `elements `using `inspect `option. Is there a way way to automatically extract the id

**Comment by buggyj** [comment]
If you hover the mouse pointer over a link when you right click (to start the clip) the link address will be copied into `@linkURL`.

**Reply by arunnbabu81** [reply]
> If you hover the mouse pointer over a link when you right click (to start the clip) the link address will be copied into `@linkURL`.

I tried this rule. It works for the links as you mentioned, but there is some problem with the code I guess because new clips are replacing the previous clips after using this code.

![image](https://user-images.githubusercontent.com/67494083/157120163-3d6a2a91-819b-4405-931b-774dac29b736.png)

**Reply by buggyj** [reply]
`$text=((*$text*))<a href="((*@linkURL..................`

**Comment by buggyj** [comment]
There is also an addon to view the anchor - https://addons.mozilla.org/en-US/firefox/addon/display-_anchors/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

**Reply by arunnbabu81** [reply]
> There is also an addon to view the anchor - https://addons.mozilla.org/en-US/firefox/addon/display-_anchors/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

This seems good. For long text without any near by links, this will be good.  How to copy the anchor value ?

**Reply by arunnbabu81** [reply]
> There is also an addon to view the anchor - https://addons.mozilla.org/en-US/firefox/addon/display-_anchors/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

I got it. This is too good an add on for this purpose. Thanks for sharing

**Comment by arunnbabu81** [comment]
This is the rule I am using now. But the newly clipped text is replacing the previous clipped text using this rule.
What might be the problem
![image](https://user-images.githubusercontent.com/67494083/157127515-c56d3359-82e9-4cdc-a495-4f0b33704e43.png)
